legion/engine/core/ecs/registry.cpp

```diff
@@ -60,11 +60,9 @@ namespace legion::core::ecs
     std::string Registry::getFamilyName(id_type id)
     {
         if (const auto itr = instance.m_familyNames.find(id); itr != instance.m_familyNames.end())
-        {
             return itr->second;
-        }
 
-        return "Component type " + std::to_string(id);
+        return "";
```

Comment:
What is this used for and is the new empty return value properly handled?

Reply:
yeah this was used, and then the empty string was used to check errors. but i'm pretty sure this can be reverted. other than that it is only used in imgui of our current makeshift editor. ultimately this fucntion should just return a result that can error out.